### DISCUSSION TITLE
OCI chart registry for Longhorn

### DISCUSSION THREAD

**Discussion by wkbrd** [longhorn/longhorn#9428] [Q&A]
Is there an OCI chart registry for the Longhorn?  The instructions mention an https:// registry, but interested in knowing if there is an oci:// registry for use with Helm.

**Comment by PhanLe1010** [comment]
I don't think Longhorn has OCI chart registry at the moment. Only the Helm HTTP repository is available now.

Is there an specific requirements/use-case that the OCI chart registry is mandatory? 

cc @derekbit

**Comment by wkbrd** [comment]
Our organization has been standardizing around the use of OCI registries over legacy Helm-based ones and therefore offering the charts in an OCI registry would directly integrate better with this approach.

**Reply by PhanLe1010** [reply]
cc @derekbit @innobead 

Maybe we can convert this one to a ticket to see if many user are interested to have Longhorn Helm OCI registry. 

As a workaround in the meantime, perhaps you can manually create your own OCI registry and push the Longhorn Helm chart there @wkbrd ?

**Comment by PhanLe1010** [comment]
Closing this discussion in favor of the github ticket https://github.com/longhorn/longhorn/issues/9450
